README.md

```diff
@@ -94,7 +94,7 @@ page by checking the [Container support check box][product-selector-container].
 The following substitutions will be used in this documentation:
 
 |                      | Meaning                                                |
-| ---------------------| :------------------------------------------------------|
+| -------------------- | :----------------------------------------------------- |
```

Comment:
Revert? See motivation in the comment on the Socket section below.

Reply:
done